docs/quickstart.md

```diff
@@ -38,14 +38,20 @@ All further instructions assumes you are in the code directory eg. `ocp4-upi-pow
 
 Update the [var.tfvars](../var.tfvars) based on your environment. Description of the variables are available in the following [link](./var.tfvars-doc.md).
 
+For sensitive variables that should not be saved on disk, set these environment variables;
+```
+$ export IBMCLOUD_API_KEY=xxxxxxxxxxxxxxx
+$ export RHEL_SUBS_USERNAME=xxxxxxxxxxxxxxx
+$ export RHEL_SUBS_PASSWORD=xxxxxxxxxxxxxxx
+```
```

Comment:
One problem with this is that it'll be in shell history which is saved in disk :-)
So best solution is to disable history and then export these variables
```
set +o history
export IBMCLOUD_API_KEY=xxxxxxxxxxxxxxx
....
set -o history
```

Reply:
True! Will be in the next push.